src/h5web/visualizer/utils.ts

```diff
@@ -1,6 +1,18 @@
 import type { HDF5Entity, HDF5Metadata } from '../providers/models';
 import { VIS_DEFS, Vis } from '../visualizations';
 
+function applySpecialVisRules(visArr: Vis[]): Vis[] {
```

Comment:
`removeUnneededVis` ? 

I wonder if this should not be kept inlined (with a ternary) to avoid the naming problem.

Reply:
👍 

I didn't want to have too much code inside the `try`/`catch`, and I thought a function could be a bit more self-documenting.